backend/common.go

```diff
@@ -122,6 +123,11 @@ func (s *DataSourceInstanceSettings) HTTPClientOptions() (httpclient.Options, er
 
 	setCustomOptionsFromHTTPSettings(&opts, httpSettings)
 
+	opts.ProxyOptions, err = s.ProxyOptions()
```

Comment:
I believe you need to do the same for `AppInstanceSettings` since now App plugins won't read the proxy configuration (which was done before in the function `parseHTTPSettings`)

Reply:
I was looking through an example where this is used for an app plugin, and I don't think this is needed, since this functionality can only live on the datasource part of the plugin. In the zabbix plugin, for example, it uses the [data souce instance settings](https://github.com/alexanderzobnin/grafana-zabbix/blob/df0ef4ba6ec3fa439a2138cb876cc23871639d90/pkg/datasource/datasource.go#L32), which is where PDC is then enabled from today.

Reply:
okay, I am just pointing the difference, if this is a feature only available for datasource plugins (or using datasource settings) then :shrug:

Reply:
For sure, I really appreciate you pointing it out! It is just a feature for datasource plugins/datasource settings :)